DataProvider/DataProvider/Model/User.swift

```diff
@@ -0,0 +1,24 @@
+//
+//  User.swift
+//  DataProvider
+//
+//  Created by İbrahim Kültepe on 11.05.2023.
+//
+
+public struct User : Decodable {
+    public let name: String?
+    public let surname: String?
+    public let followedCount: Int
+    public let recipeCount: Int
+    public let userImage: UserImage
```

Comment:
```suggestion
    public let image: UserImage
```

User objesi içinde direk image diyebiliriz